test/library/packages/UnrolledLinkedList/indexOf/testIndexOf.chpl

```diff
@@ -0,0 +1,27 @@
+private use UnrolledLinkedList;
+
+config const testIters = 120;
+
+var lst: unrolledLinkedList(int, true);
+var idx = 0;
+
+for i in 1..testIters {
+  lst.append(i);
+  idx = lst.indexOf(i);
+  assert(idx == i-1);
+}
```

Comment:
I think what Paul suggests below can be achieved by breaking this loop into two where the first one appends and the second checks the index of elements. Currently you are always searching for the element that's at the end of the list. Splitting the loop will make it look at elements all around the list, including what he suggests, I think.

Reply:
At the time I made my cocmment below, `testIters` above was 8, so there was only ever one node.  Bumpiing that up was all I had in mind, but the change you suggest would be even better.

Reply:
You're right. I've made the change.